docs/conf.py

```diff
@@ -296,3 +297,79 @@ def setup(app):
 if read_the_docs_build:
 
     subprocess.call('cd ../minorminer/docs/; make cpp', shell=True)
+
+github_map = {'dwavebinarycsp': 'dwavebinarycsp',
+              'cloud': 'dwave-cloud-client',
+              'dimod':  'dimod',
+              'dwave_networkx': 'dwave-networkx',
+              'greedy': 'dwave-greedy',
+              'hybrid': 'dwave-hybrid',
+              'inspector': 'dwave-inspector',
+              'minorminer': 'minorminer',
+              'neal': 'dwave-neal',
+              'penaltymodel': {'cache': 'penaltymodel_cache',
+                               'core': 'penaltymodel_core',
+                               'lp': 'penaltymodel_lp',
+                               'maxgap': 'penaltymodel_maxgap',
+                               'mip': 'penaltymodel_mip'},
+              'system': 'dwave-system',
+              'embedding': 'dwave-system',
+              'tabu': 'dwave-tabu'}
+
+def linkcode_resolve(domain, info):
+    """
+    Find the URL of the GitHub source for dwave-ocean-sdk objects.
+    """
+    # Based on https://github.com/numpy/numpy/blob/main/doc/source/conf.py
```

Comment:
Probably enough code that we need to include a copy of their [LICENSE](https://github.com/numpy/numpy/blob/main/LICENSE.txt).

Reply:
@silviacorbellaro, see https://github.com/dwavesystems/dwave-ocean-sdk/pull/121/commits/90589c1e270cc3c6f54e25107d062d6ae437796f based on our discussion.

Reply:
FWIW, I think we have historically just included it as a comment in the code.